README.md

```diff
@@ -1,35 +1,130 @@
-# Membrane Template Plugin
+# Membrane Style Transfer Plugin
 
-[![Hex.pm](https://img.shields.io/hexpm/v/membrane_template_plugin.svg)](https://hex.pm/packages/membrane_template_plugin)
-[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_template_plugin)
-[![CircleCI](https://circleci.com/gh/membraneframework/membrane_template_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_template_plugin)
+[![Hex.pm](https://img.shields.io/hexpm/v/membrane_style_transfer_plugin.svg)](https://hex.pm/packages/membrane_style_transfer_plugin)
+[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_style_transfer_plugin)
+[![CircleCI](https://circleci.com/gh/membraneframework/membrane_style_transfer_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_style_transfer_plugin)
 
-This repository contains a template for new plugins.
+This repository contains `Membrane.StyleTransfer` - `Membrane` filter performing style transfer on raw video frames.
 
-Check out different branches for other flavors of this template.
+It uses [Ortex](https://github.com/elixir-nx/ortex) to run models serialized in `.onnx` format and [Nx](https://github.com/elixir-nx/nx) to perform data pre- and postprocessing. 
 
 It's a part of the [Membrane Framework](https://membrane.stream).
 
 ## Installation
 
-The package can be installed by adding `membrane_template_plugin` to your list of dependencies in `mix.exs`:
+The package can be installed by adding `membrane_style_transfer_plugin` to your list of dependencies in `mix.exs`:
 
 ```elixir
 def deps do
   [
-    {:membrane_template_plugin, "~> 0.1.0"}
+    {:membrane_style_transfer_plugin, github: "membraneframework-labs/membrane_style_transfer_plugin"}
   ]
 end
 ```
 
 ## Usage
 
-TODO
+Here we have 2 simple examples of usage `Membrane.StyleTransfer`. Both of them take video from the camera device available in your computer, perofrm style transfer and present the output video in the player.
+
+On the beggining, install necessary dependencies.
+
+```elixir
+Mix.install([
+  {:membrane_style_transfer_plugin, github: "membraneframework-labs/membrane_style_transfer_plugin"},
+  {:membrane_camera_capture_plugin, "~> 0.7.2"},
+  {:membrane_ffmpeg_swscale_plugin, "~> 0.15.1"},
+  {:membrane_sdl_plugin, "~> 0.18.2"}
+])
+```
+
```

Comment:
Some text comment could be helpful here, something like: "Then, run the following script":